.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,71 @@
+---
+name: "🐛 Bug Report"
+description: Create a report to help us improve
+title: "(short issue description)"
+labels: [bug]
+assignees: []
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: What is the problem? A clear and concise description of the bug.
+    validations:
+      required: true
+
+  - type: textarea
+    id: current
+    attributes:
+      label: Current Behavior
```

Just to be consistent with the other fields
```suggestion
      label: Current behavior
```